docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.7'
+services:
+  frontend:
+    build: frontend
+    ports:
+      - '3000:3000'
```

Comment:
We should not run Flotilla on `3000` as that is the port used for ISAR. I'll make an issue to change port.

Reply:
See #57